target/linux/ramips/dts/kn_rc.dts

```diff
@@ -0,0 +1,129 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "ZyXEL Keenetic Omni";
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wan {
+			label = "kn_rc:green:wan";
+			gpios = <&gpio0 38 GPIO_ACTIVE_LOW>;
+		};
+
+		usb {
+			label = "kn_rc:green:usb";
+			gpios = <&gpio0 39 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi {
+			label = "kn_rc:green:wifi";
+			gpios = <&gpio0 72 GPIO_ACTIVE_LOW>;
+		};
+
+		power {
+			label = "kn_rc:green:power";
+			gpios = <&gpio0 20 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		fn {
+			label = "fn";
+			gpios = <&gpio0 17 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+		};
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "w25q128";
```

Comment:
According to the [kernel documentation](http://lxr.free-electrons.com/source/Documentation/devicetree/bindings/mtd/jedec,spi-nor.txt?v=4.4):

> May include a device-specific string consisting of the manufacturer **and** name of the chip... **Must** also include "jedec,spi-nor" for any SPI NOR flash that can be identified by the JEDEC READ ID opcode (0x9F).

Long story short, use ``compatible = "jedec,spi-nor";`` only!